.github/wiki/Getting-Started.md

```diff
@@ -3,12 +3,12 @@
 The CLI can be installed globally or run directly with `npx`:
 
 ```bash
-npm install -g @nimblehq/infra-template
+npm install -g @nimblehq/infra-template@latest
```

Comment:
Just a question, if we already have an older version installed, will this command update the package? If not, is it better to recommend only `npx` (or to add a `ensure you have the latest version)? (it's really bad if someone starts a project with an old version and realize it too late ^^)

Reply:
Yes, this command can update the package to the specific version (in this case, it will fetch the latest version to install)